csharp/ReadJsonFile.cs

```diff
@@ -0,0 +1,27 @@
+// Read JSON file, transform and print to stdout
+using System;
+using System.IO;
+using System.Text.Json;
+
+public class ReadJsonFile{
+    public static void Main(string[] args){
+        string jsonFilePath = args[0];
+        string jsonContent = File.ReadAllText(jsonFilePath);
+
+        JsonDocument doc = JsonDocument.Parse(jsonContent);
+        JsonElement root = doc.RootElement;
```

Comment:
```suggestion
        JsonElement root = JsonDocument.Parse(jsonContent).RootElement;
```

Can combine into one line b/c we don't use the `doc` variable anywhere else.